src/reporters/junit.ts

```diff
@@ -80,7 +80,7 @@ export default class JUnitReporter extends BaseReporter {
   override onStepEnd(
     journey: Journey,
     step: Step,
-    { status, error, start, end }: StepEndResult
+    { }: StepEndResult
```

Comment:
should this be removed if it's not needed, are you keeping it for backward compatibility?

Reply:
This can be removed, forgot to clean up.